### ISSUE TITLE
#1296 - PnP search filter is not accessible when using tab key.

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1650]
#1296 - PnP search filter is not accessible when using tab key. I am using PnP search package 4.5.0 and this issue is not fixed in the released package.

_Originally posted by @wobba in https://github.com/microsoft-search/pnp-modern-search/issues/1296#issuecomment-943386077_

**Comment by wobba** [comment]
Should be readded now in 4.5.2

**Comment by MariaAug** [comment]
Great. Thanks !!

**Comment by MariaAug** [comment]
Hi @wobba , Looks like the actual issue is not fixed yet though I can see now role and aria label added in 4.5.2 package. This issue was related to PnP search filter not accessible when using tab key.

**Comment by wobba** [comment]
I did bring over the code from the last PR. I'll re-open this and address it for v4.6

**Comment by MariaAug** [comment]
@wobba - Thanks. I cleared the cached and inspected the filter element. It shows <span role="button" aria-label="Filters">Filters</span>.  I think tabindex attribute is missing which could be the reason the focus is not moving to the filter icon using tab key?

**Comment by wobba** [comment]
I'm adding it back now but there's a new control in filter to allow filtering on values and clearing the selected one. This is UI fabric control and seems it does not work well with keyboard navigation. I'll see if I can figure it out and feel free to comment how navigation ideally should work on filters with the different filter templates.

**Comment by CollinHunterGOC** [comment]
This seems to be still be an issue in version 4.7.0. Has this been re-addressed in later versions?

**Comment by wobba** [comment]
@CollinHunterGOC  can you log a new issue for the latest version of the web parts and describe what is missing in the markup to make this work?
